pkg/mls/store/store.go

```diff
@@ -53,6 +64,111 @@ func New(ctx context.Context, config Config) (*Store, error) {
 	return s, nil
 }
 
+func (s *Store) PublishIdentityUpdate(ctx context.Context, req *identity.PublishIdentityUpdateRequest) (*identity.PublishIdentityUpdateResponse, error) {
+	new_update := req.GetIdentityUpdate()
+	if new_update == nil {
+		return nil, errors.New("IdentityUpdate is required")
+	}
+
+	// TODO: Implement serializable isolation level once supported
+	if err := s.db.RunInTx(ctx, &sql.TxOptions{ /*Isolation: sql.LevelSerializable*/ }, func(ctx context.Context, tx bun.Tx) error {
```

Comment:
Why is every tool for working with SQL in Go so terrible?

Reply:
I hate Bun. I hate Gorm. Sqlx is fine but very bare-bones.

I've heard good things about [`sqlc` but never tried it](https://sqlc.dev/)

Reply:
Awesome, this will be a good list to try when we come back to this. Always hate working with ORM's